components/about.js

```diff
@@ -1,10 +1,23 @@
 import React from 'react';
+import EmailForm from './email-form';
 
 export default class AboutComponent extends React.Component {
+  constructor(props) {
+    super(props);
+    const { success, errors, value } = this.props.location.query;
+    if (success) {
+      this.emailFormProps = {
+        success: success === "true",
+        errors: errors && errors.split(',') || [],
+        value
+      }
```

Comment:
when the form is submitted on the server, the URL query params contain info about the result: `/about?succes=true&value=jack@jack.com`, and this code tidies them and passes them down to `EmailForm`